### PR TITLE
BCN - 04/19 - Arnau

### PR DESCRIPTION
**Pull request by abeumala**
:)

### CODE REVIEW

File: src/clue.js [hunk 1]
```diff
@@ -84,11 +84,182 @@ name: Guest House
 name: Patio
 */
 
+// Characters
+
+ var mrGreen = {
+	first_name: "Jacob",
```
Comment (tawebbcn) [inline src/clue.js:90]: Try to use  ' ' for javascript and " " for HTML

File: src/clue.js [hunk 2]
```diff
@@ -84,11 +84,182 @@ name: Guest House
...
+
+}
+
+
+function revealMistery(envelope) { //misteryEnvelope as a parameter
+	alert(envelope[0].first_name + " " + envelope[0].last_name + " killed Mr.Boddy using a " + envelope[0].color + " " + envelope[2].name + " in the " + envelope[1].name + "!!!!");
+}
+
+
+
```
Comment (tawebbcn) [inline src/clue.js:242]: What is this???

File: src/clue.js [hunk 3]
```diff
@@ -84,11 +84,182 @@ name: Guest House
...
+// weaponsArray.push(study)
+// weaponsArray.push(rope)
+// weaponsArray.push(rope)
+// weaponsArray.push(rope)
+// weaponsArray.push(rope)
+// weaponsArray.push(rope)
+
+
+function randomSelector(deck) {
+	var randomIndex = Math.floor(Math.random()*deck.length);
```
Comment (tawebbcn) [inline src/clue.js:222]: Fix indentation, I have the feeling you are using 4 spaces and not 2.

**Review by tawebbcn — COMMENTED** [review]
Good job, but be careful with the format.

File: src/clue.js (hunk 3 above)
Comment (abeumala) [reply src/clue.js:222]: in my text processor i had 2 spaces and it did not show like this idk :P but i'll take a look

**Pull request closed without merging**